woocommerce/woocommerce-android
# Enable Gradle Cache

### Description


This PR enables Gradle Cache by default (keeps it disabled on CI). It should speed up local build times - especially after switching branches. More info https://docs.gradle.org/current/userguide/build_cache.html

### Testing instructions


Run eg. ` ./gradlew woocommerce:assembleWasabiDebug --scan` and make sure Gradle Cache is enabled.
